docs/developers/connecting-to-ethermint.md

```diff
@@ -0,0 +1,30 @@
+# Connecting Metamask to Ethermint
+
+In this guide you'll connect your wallet to the Ethermint fork RPC Node.
+
+## Prerequisites
+
+If you do not have [MetaMask](https://metamask.io/) installed, please install it
+[here](https://metamask.io/download/)
+
+## Configuring the network details
+
+First, open the MetaMask settings and choose "Add Network"
+
+![Configuring MetaMask](../../static/img/add-ethermint.png)
+
+Next, enter the settings as follows:
+
+__Network name__: Ethermint on Celestia
+
+__RPC URL__: `http://35.208.160.145:8545`
+
+__Chain ID__: 69420
+
+__Currency__: ETC
```

Comment:
Are you sure we want to use `ETC` here? 😂

Reply:
ah right 😅 How about CET or CESE? Or any suggestions?

Reply:
Either of those work!